packages/cli/lib/projects.js

```diff
@@ -121,19 +120,18 @@ const createProjectConfig = async (projectPath, projectName, template) => {
     }`
   );
 
-  if (template === 'no-template') {
+  if (template.name === 'no-template') {
```

Comment:
I think you'll need to update this here for this to work https://github.com/HubSpot/hubspot-cli/blob/master/packages/cli/commands/project/download.js#L61

Reply:
Ooh, thanks so much for catching this!

Reply:
Ooh, thanks so much for catching this!